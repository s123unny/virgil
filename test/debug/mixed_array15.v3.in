b ../core/mixed_array15.v3 3
b ../core/mixed_array15.v3 12
run
bt
info l
c
bt
info l
c
q
