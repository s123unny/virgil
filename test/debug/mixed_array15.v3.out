breakpoint|../core/mixed_array15.v3 3|0
breakpoint|../core/mixed_array15.v3 12|1
stop|stopOnBreakpoint
bt|main|../core/mixed_array15.v3|12
variable|0|a|0|int|false
variable|1|e|(141235, #1:Bar:Bar.m)|(int, int -> int)|false
stop|stopOnBreakpoint
bt|Bar.m|../core/mixed_array15.v3|3
bt|main|../core/mixed_array15.v3|12
variable|0|this|#1:Bar|Bar|true
variable|1|a|141235|int|false
stop|end
result|141243
